<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPortMappingDialogUi</class>
 <widget class="QDialog" name="AddPortMappingDialogUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>174</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Port Mapping</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="descriptionLabel">
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="descriptionLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="externalPortLabel">
       <property name="text">
        <string>External Port</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="externalPortLineEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="protocolLabel">
       <property name="text">
        <string>Protocol</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="protocolComboBox">
       <item>
        <property name="text">
         <string>TCP</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>UDP</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Both</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="internalIPAddressLabel">
       <property name="text">
        <string>Internal IP address</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="internalIPAddressLineEdit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="internalPortLabel">
       <property name="text">
        <string>Internal Port</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="internalPortLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddPortMappingDialogUi</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddPortMappingDialogUi</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
